tests/ci/android/AWSLCAndroidTestRunner/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# For more information about using CMake with Android Studio, read the
+# documentation: https://d.android.com/studio/projects/add-native-code.html
+
+# Sets the minimum version of CMake required to build the native library.
+
+cmake_minimum_required(VERSION 3.4.1)
+
+# AWS lib
+set(path_to_awslc "${CMAKE_CURRENT_LIST_DIR}/../../../../../../../../")
+get_filename_component(path_to_awslc ${path_to_awslc} ABSOLUTE)
+
+list(APPEND CMAKE_MODULE_PATH "${path_to_awslc}")
```

Comment:
Can this just use the `crypto` target? If you need the path to it you can use `$<TARGET_FILE:crypto>`